packages/components/src/components/tabs-bar/tabs-bar.tsx

```diff
@@ -89,6 +90,7 @@ export class TabsBar {
   private scrollerElement: HTMLPScrollerElement;
   private direction: ScrollerDirection = 'next';
   private hasPTabsParent: boolean;
+  private tabsAreButtons: boolean;
```

Comment:
Perhaps rename since booleans usually start with the verb.
```suggestion
  private areTabsButtons: boolean;
```